ui/client/components/graph/node-modal/NodeDetailsContent.js

```diff
@@ -500,7 +500,9 @@ export class NodeDetailsContent extends React.Component {
                   showValidation={showValidation}
                   validators={validators}
                   onChange={handleChange}
-                  className={(!showValidation || allValid(validators, [fieldValue])) ? "node-input" : "node-input node-input-with-error"}/>
+                  className={(!showValidation || allValid(validators, [fieldValue])) ? "node-input" : "node-input node-input-with-error"}
+                  placeholder={""} // FIXME: what should be here?
```

Comment:
Do we need placeholder?

Reply:
don't know - it is used in lower level in concrete fields